### DISCUSSION TITLE
[Unstable] Console Limb bug and Husks are Influenced by the Miracle Worker medic talent

### DISCUSSION THREAD

**Discussion by Holneer** [FakeFishGames/Barotrauma#14664] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Whenever I get hit by husks (both new and old) I get console error pop-ups.
Miracle worker works on husks

### Reproduction steps

Issue 1: Randomly whenever I get hit by husks (both the new and the old).
![image](https://github.com/user-attachments/assets/a99ab2f4-a308-4701-9fcf-164f91f7bdaa)

Issue 2: Playing as a Medic and having the Miracle Worker talent prevents the Husks from dying (only tested while wearing Zealot Robes). I'm guessing I'm being flagged as part of their faction/team when I wear the cultist stuff so the talent starts afflicting them making it impossible for them to die for as long as I stay close.

![602960_20240920214735_1](https://github.com/user-attachments/assets/e2f2b983-b379-4a39-aae5-55f461859fde)

![602960_20240920214732_1](https://github.com/user-attachments/assets/253ffebd-357b-40d4-b6f1-5ab291a968d6)

Stood for several minutes next to them and they only died when I went into a different room (exceeded talent's range).

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I unfortunately couldn't find a way to reproduce the console error. The error seems to imply that the character attempted to attack using a limb that the client doesn't know about - so I suspect they'd gained the "husk stringer" server-side, but somehow not client-side.

**Comment by Regalis11** [comment]
The 2nd issue should now be fixed in our private repo, fix coming up in the next unstable build and the upcoming release.
